app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -2,7 +2,8 @@
 // Also handles the sidebar and tabs logic.
 import React from "react";
 import cx from "classnames";
-import { get } from "lodash/fp";
+import Cookies from "js-cookie";
```

Comment:
Why cookies? No cookies please.

Reply:
I realize now it was there before... but good opportunity to change.

Reply:
I'd prefer to see us refactor all the report page cookie state at once.

Reply:
I'll put it on my todo list.